lib/addons/src/index.js

```diff
@@ -2,7 +2,7 @@ export class AddonStore {
   constructor() {
     this.loaders = {};
     this.panels = {};
-    this.channel = null;
+    this.channel = { on() {}, emit() {} };
```

Comment:
@ndelangen completely disagree. if this doesn't get overwritten it will cause an error to fail silently! having been bit by this bug before, it will be even harder to debug when there is a problem!